tests/cross_validation/c_harness/README.md

```
@@ -0,0 +1,95 @@
+JAX_ENABLE_X64=1 PYTHONPATH=/path/to/ripple/src \
+    python compare.py earth00-40-DE405.dat.gz sun00-40-DE405.dat.gz \
+        out_exact.bin out_gen0.bin out_genhet.bin
+```
+
+## Reference result
+
+The figure of merit is the **overlap loss** (mismatch) `1 − ⟨h₁|h₂⟩/√(⟨h₁|h₁⟩⟨h₂|h₂⟩)`
+between the reconstructed strain and the compiled-LAL strain — the same metric
+(and numerically-stable form) used by `test_lal_overlap.py`, reported as `log10`.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)